resources/prosody-plugins/mod_short_lived_token.lua

```diff
@@ -0,0 +1,126 @@
+-- to be enabled under the main virtual host with all required settings
+-- short_lived_token = {
+--     issuer = 'myissuer';
+--     accepted_audiences = { 'file.sharing' };
+--     key_path = '/etc/prosody/short_lived_token.key';
```

Comment:
What type of key is this? Would be good to document for those who intend to deploy this.